app/src/main/res/layout/activity_benchmark_add_photo.xml

```diff
@@ -16,10 +16,10 @@
         android:layout_marginTop="156dp"
         android:layout_marginEnd="4dp"
         android:background="#00000000"
-        app:icon="@drawable/right"
+        app:icon="@drawable/right_array"
         app:iconTint="@color/black"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.438"
+        app:layout_constraintHorizontal_bias="0.41"
```

Comment:
Why are we relying on some random value? Change to dp, or make relative to other components.

Reply:
https://www.google.com/search?q=what+is+bias+in+constraint+layout+android&oq=what+is+constraint+bias&aqs=chrome.1.69i57j0i22i30l2.6492j0j7&sourceid=chrome&ie=UTF-8